Python-Using-Pip.md

```diff
@@ -0,0 +1,34 @@
+# Python Using Pip
+
+We had seen about how to use `import` statements to `import` various modules and to use them in our programs. Python itself comes with several built-in modules. But python community has more to offer.
+> Its the modules that makes python so powerful!
+
+Third party module has added so much more functionality to Python. Now we would learn how to install these modules so that we can use those in our programs.
```

Comment:
~~Third party module has added so much more functionality to Python.~~ Third party modules add so much more functionality to Python.